package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "shelley",
+  "version": "0.1.1-alpha.1",
+  "license": "TBC",
+  "description": "UI Component lib... Yes; another one.",
+  "homepage": "https://github.com/action-is-hope/shelley",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/action-is-hope/shelley.git"
+  },
+  "keywords": [
+    "gatsby, typescript"
+  ],
+  "author": "Action is Hope<info@action-is-hope.earth>",
+  "dependencies": {
+    "@stylable/core": "1.0.5",
+    "classnames": "^2.2.6",
+    "gatsby": "^2.18.4",
+    "gatsby-link": "^2.2.25",
+    "gatsby-plugin-react-helmet": "^3.1.16",
+    "gatsby-plugin-stylable": "^1.1.1",
+    "gatsby-plugin-typescript": "^2.1.20",
+    "react-dom": "^16.10.1",
+    "react-helmet": "^5.2.0"
+  },
+  "main": "n/a",
+  "scripts": {
+    "type-check": "tsc",
```

Comment:
Another ticket for the component build stuff I think so that we can store this setup as our base boilerplate...

Reply:
It would be nice to verify a build as part of this essential story zero? Maybe...

Reply:
Maybe get something basic in having said that. It would be nice to finish this story with a novel release to npm.

Reply:
I don' think we want the build stuff in before we fork it as a starter kit. We can add a ticket for the build stuff as previously suggested.